src/components/SelectionList/types.ts

```diff
@@ -263,6 +263,9 @@ type ListItemProps<TItem extends ListItem> = CommonListItemProps<TItem> & {
 
     /** Whether to show RBR */
     shouldDisplayRBR?: boolean;
+
+    /** Styles for the wrapper view */
+    wrapperStyle?: StyleProp<ViewStyle>;
```

Comment:
Remove. This is already included in `CommonListItemProps`

Reply:
Updated.